src/routes/host/components/InstanceManagerComponent.js

```diff
@@ -12,13 +12,12 @@ class InstanceManagerComponent extends React.Component {
     const currentNode = this.props.currentNode
 
     let data = []
-    let engineimageObj = { name: 'Engine Image', image: 'N/A', state: 'N/A', id: 'engineRowKey' }
+    let engineimageObj = { name: 'Engine Image', image: defaultEngineImage.value, state: 'not deployed', id: 'engineRowKey' }
     let instanceManagerObj = { name: 'Instance Manager', image: 'N/A', state: 'N/A', id: 'instanceRowKey' }
 
     engineimage.forEach((item) => {
```

Comment:
The question is about the existing code instead of the change. 

Why we use `.forEach` in this case? because I assume `engineimage` should only have one element of data list or we just use `.some` to check.

Reply:
Using `some` does reduce the number of loops. I thought the engine data would not be very large, so I used `foreach`.

` engineimage.some((item) => {
      if (defaultEngineImage.value === item.image && item.nodeDeploymentMap && item.nodeDeploymentMap[currentNode.id]) {
        engineimageObj.state = 'deployed'
        return true
      }
      return false
    })
`
Thank you for the reminder I will pay attention to the use of my cycle

Reply:
NP, I know your intention, just double check. Thanks.